src/components/docs/Buckets/LeftIconBucket/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
There are at least 2 components that have an icon on the left. Let's make the class name more clear.